web/src/components/tabs/Tab/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { useTheme } from 'react-jss'
+
+import { Text } from '../../typography'
+
+import useStyles from './styles'
+
+const getTextSize = (size): 'body-large' | 'body-base' | 'body-small' => {
+  switch (size) {
+    case 'large':
+      return 'body-large'
+    case 'regular':
+      return 'body-base'
+    case 'small':
+      return 'body-small'
+    default:
+      return 'body-base'
+  }
+}
```

Comment:
Ok for now -> we will change `Text` size prop to 'small'|'regular'|'large' aswell!

Will do this in a separate PR after things are merged